### PR TITLE
Incorporating the Muon Veto into REDAX

### PR DESCRIPTION
**Pull request by AlexElykov**
A diagram and a general note about the Muon Veto DAQ changes for XENONnT can be found here: [Muon Veto XENONnT](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:alexelykov:m_veto). These changes were already tested on reader5, would be nice if we could further test them while actually reading data from the digitisers and in a linked mode with the TPC.

Changes:
1. Makefile: Added V1495 class
2. V1495.cc/hh: class to handle register writing to the V1495 board. Maybe it can be later used to operate the TPC/n_veto V1495 boards as well 
3.  CControl_Handler.cc/hh:
    - Some extra logic to arm/init only the devices which were specified in the config
    - Control over the Muon Veto V1495  
    - Added V1495 register values output to status doc
4. V1724_MV.cc/hh: Added a different register definition for setting the Nth channel threshold

### CODE REVIEW

File: CControl_Handler.cc [hunk 1]
```diff
@@ -57,29 +57,63 @@ int CControl_Handler::DeviceArm(int run, Options *opts){
...
+ 
+  // Getting options for the Muon Veto V1495 board
+  // Init V1495_MV only when included in config - Muon Veto only
+  std::vector<BoardType> mv = fOptions->GetBoards("V1495", fProcname);
+  BoardType mv_def = mv[0];
+  fBID = mv_def.board;
+  if (mv.size() == 1){
+     	fV1495 = new V1495(fLog, fOptions, mv_def.board, fBoardHandle, mv_def.vme_address);
+	// Writing registers to the V1495 board
+	for(auto regi : fOptions->GetRegisters(fBID)){
```
Comment (darrylmasson) [inline CControl_Handler.cc]: What do we do if a V1495 has the same SN as a V1724? Might future-proof us to add a GetV1495Opts routine.

File: CControl_Handler.cc [hunk 2]
```diff
@@ -166,14 +200,21 @@ bsoncxx::document::value CControl_Handler::GetStatusDoc(std::string hostname){
...
+  if(fV1495 != NULL){
+     in_array << bsoncxx::builder::stream::open_document
+	      << "type" << "V1495"
+	      << "Module reset" << fOptions->GetRegisters(fBID)[0].val
+	      << "Mask A" << fOptions->GetRegisters(fBID)[1].val
+	      << "Mask B" << fOptions->GetRegisters(fBID)[2].val
+	      << "Mask D" << fOptions->GetRegisters(fBID)[3].val
+	      << "Majority Threshold" << fOptions->GetRegisters(fBID)[4].val
+	      << "Coincidence Window" << fOptions->GetRegisters(fBID)[5].val
+	      << "NIM/TTL CTRL" << fOptions->GetRegisters(fBID)[6].val
```
Comment (darrylmasson) [inline CControl_Handler.cc]: Rather than 7 calls to GetRegisters, might be better to call once and cache the return.

File: V1724_MV.hh [hunk 3]
```diff
@@ -10,7 +10,7 @@ public:
   virtual ~V1724_MV();
 
 private:
-
+  unsigned int fChTrigRegister;
```
Comment (darrylmasson) [inline V1724_MV.hh]: Why is this here? This is on L54 of V1724.hh.

**Comment by AlexElykov** [comment]
Forgot to add the GetHandle in V2718 in the original PR. 
PS. recompiled all including the changes from https://github.com/coderdj/redax/pull/46 - it compiles fine, I guess as we discussed we can test it at LNGS next week.

File: CControl_Handler.hh [hunk 4]
```diff
@@ -24,11 +24,13 @@ public:
...
 
   V2718 *fV2718;
-  V1495 *fV1495;
   DDC10 *fDDC10;
+  V1495 *fV1495;
 
   int fStatus;
   int fCurrentRun;
+  int fBID;
+  int fBoardHandle;
```
Comment (darrylmasson) [inline CControl_Handler.hh]: Is there a need to make these members rather than locals? I don't see them used anywhere else.

Comment (AlexElykov) [reply CControl_Handler.hh]: > Is there a need to make these members rather than locals? I don't see them used anywhere else.

Not really. I guess they are just leftovers from some other variation of the code.

**Comment by darrylmasson** [comment]
This PR fixes #7
